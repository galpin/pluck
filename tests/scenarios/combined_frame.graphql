{
  cores(limit: 5) {
    id
    status
    missions @frame {
      name
      flight
    }
  }
}
