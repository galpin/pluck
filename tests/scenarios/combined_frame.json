{
  "data": {
    "cores": [
      {
        "id": "B1015",
        "status": "lost",
        "missions": [
          {
            "name": "CRS-6",
            "flight": 22
          }
        ]
      },
      {
        "id": "B0006",
        "status": "lost",
        "missions": [
          {
            "name": "CRS-1",
            "flight": 9
          }
        ]
      },
      {
        "id": "B1034",
        "status": "lost",
        "missions": [
          {
            "name": "Inmarsat-5 F4",
            "flight": 40
          }
        ]
      },
      {
        "id": "B1016",
        "status": "lost",
        "missions": [
          {
            "name": "T\u00fcrkmen\u00c4lem 52\u00b0E / MonacoSAT",
            "flight": 23
          }
        ]
      },
      {
        "id": "B1025",
        "status": "inactive",
        "missions": [
          {
            "name": "CRS-9",
            "flight": 32
          },
          {
            "name": "Falcon Heavy Test Flight",
            "flight": 55
          }
        ]
      }
    ]
  }
}