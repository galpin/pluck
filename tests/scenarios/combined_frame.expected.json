{
 "missions": [
  {
   "name": "CRS-6",
   "flight": 22
  },
  {
   "name": "CRS-1",
   "flight": 9
  },
  {
   "name": "Inmarsat-5 F4",
   "flight": 40
  },
  {
   "name": "T\u00fcrkmen\u00c4lem 52\u00b0E / MonacoSAT",
   "flight": 23
  },
  {
   "name": "CRS-9",
   "flight": 32
  },
  {
   "name": "Falcon Heavy Test Flight",
   "flight": 55
  }
 ]
}